unitary/examples/quantum_rpg/item.py

```diff
@@ -58,8 +59,12 @@ def get_action(self, user_input: str) -> Optional[ITEM_ACTION_TYPE]:
             if isinstance(targets, str):
                 targets = [targets]
             if keyword == keywords or keyword in keywords:
-                if not targets or target in targets:
-                    return action
+                if isinstance(targets, re.Pattern):
```

Comment:
Would be more general to allow a sequence of patterns or a mix of patterns and literals in `targets`. Won't add too much complexity to the code from what I see.